Add fingerprinting docs (#995)
Macrometacorp/docs: docs/_photoniq/fingerprinting/fingerprinting-metrics.md

---
sidebar_position: 40
title: Fingerprinting Metrics
---

> Digital Fingerprinting provides visibility into fingerprinting performance by providing the tracking metrics listed below. These metrics quantify the scale of fingerprinting activity and can help identify usage trends.

Reviewer: ```suggestion
The tracking metrics below quantify the scale of the Digital Fingerprinting service and can help identify usage trends.
```

## Stability Rate

> Percentage of visits where an existing visitor was recognized using fingerprints. This indicates how often FPS can recognize visitors persistently.

Reviewer: ```suggestion
Percentage of visits where a previous visitor was recognized by the fingerprinting service (FPS). This indicates how often FPS can recognize visitors persistently.
```

## Unique Visitor Count

> The total number of unique visitors identified by FPS fingerprints over a period.

Reviewer: ```suggestion
The total number of unique visitors identified by FPS over a period.
```

## Request Count

> Total number of fingerprinting requests received per minute.

Reviewer: This says fingerprinting requests  received per minute, but the metric below lists requests per second. - Would this not say Requests per Minute or os this supposed to be Total Requests?

Author: This is literally what the engineers named it. @joubert-macrometa, could you take a look, please?

Reviewer 2: We received 10,000 requests this past minute.
We received 166 requests per second this past minute.

Technically, they say the same thing:

166*60seconds ~ 10,000

But I also noticed something different:
Request counts are how many we _received_.
RPS is how many are _served_.

I think the received/served was a wording mistake, but they technically mean different things.
Durga wrote this originally, so let's ask him to clarify.

Author: @joubert-macrometa, any word?

Reviewer 2: You fixed it already.  We removed one  of these.
